concordium-consensus/tests/scheduler/SchedulerTests/Helpers.hs

```diff
@@ -123,7 +142,7 @@ forEveryProtocolVersion check =
 -- |Construct a test block state containing the provided accounts.
 createTestBlockStateWithAccounts ::
     (Types.IsProtocolVersion pv) =>
-    [Basic.Account (Types.AccountVersionFor pv)] ->
+    [BS.PersistentAccount (Types.AccountVersionFor pv)] ->
```

Comment:
You could also have an extra helper that takes `[PersistentBSM pv (PersistentAccount ...)]` which would do

```
createTest... accs = createTestBlockStateWithAccounts =<< sequence accs
```

That might make it a tiny bit less awkward to use.